docs/core/resilience/http-resilience.md

```diff
@@ -81,13 +81,26 @@ The preceding code adds the standard resilience handler to the <xref:System.Net.
 
 The default configuration chains five resilience strategies in the following order (from the outermost to the innermost):
 
-| Order | Strategy | Description |
-|--:|--|--|
-| **1** | Rate limiter | The rate limiter pipeline limits the maximum number of concurrent requests being sent to the dependency. |
-| **2** | Total request timeout | The total request timeout pipeline applies an overall timeout to the execution, ensuring that the request, including retry attempts, doesn't exceed the configured limit. |
-| **3** | Retry | The retry pipeline retries the request in case the dependency is slow or returns a transient error. |
-| **4** | Circuit breaker | The circuit breaker blocks the execution if too many direct failures or timeouts are detected. |
-| **5** | Attempt timeout | The attempt timeout pipeline limits each request attempt duration and throws if it's exceeded. |
+| Order | Strategy | Description | Defaults |
+|--:|--|--|--|
+| **1** | Rate limiter | The rate limiter pipeline limits the maximum number of concurrent requests being sent to the dependency. | Queue: `0`<br>Permit: `1_000` |
+| **2** | Total timeout | The total request timeout pipeline applies an overall timeout to the execution, ensuring that the request, including retry attempts, doesn't exceed the configured limit. | Total timeout: 30s |
+| **3** | Retry | The retry pipeline retries the request in case the dependency is slow or returns a transient error. | Max retries: `3`<br>Backoff: `Exponential`<br>Use jitter: `true`<br>Delay:2s |
+| **4** | Circuit breaker | The circuit breaker blocks the execution if too many direct failures or timeouts are detected. | Failure ratio: 10%<br>Min throughput: `100`<br>Sampling duration: 30s<br>Break duration: 5s |
+| **5** | Attempt timeout | The attempt timeout pipeline limits each request attempt duration and throws if it's exceeded. | Attempt timeout: 10s |
+
+#### Retries and circuit breakers
+
+The retry and circuit breaker strategies both handle a set of specific HTTP status codes and exceptions. Consider the following HTTP status codes:
+
+- HTTP 500 and above (Server errors)
+- HTTP 408 (Request timeout)
+- HTTP 429 (Too many requests)
+
+Additionally, these strategies handle the following exceptions:
+
+- `HttpRequestException`
+- `TimeoutRejectedException`
```

Comment:
Can we do the same thing for hedging? Just add circuit breaker exception too.

Reply:
I'm not sure I follow. So, the Hedging handler employs what defaults?

Reply:
Hedging has different `ShouldHandle` predicate compared to standard resilience:

https://github.com/dotnet/extensions/blob/b60b982e9b5bc6273aaed1f70f9002e381aae97a/src/Libraries/Microsoft.Extensions.Http.Resilience/Hedging/HttpClientHedgingResiliencePredicates.cs#L53

Maybe just note that status codes/exceptions are the same as compared to standard resilience, but hedging strategy also handles `BrokenCircuitException`.